### PR TITLE
chore: Added Calci

### PR DESCRIPTION
**Pull request by SyedImtiyaz-1**
## Description
Here is a Beginner friendly project Calci 👍🏻 
This Project uses JS for click events and the code base is easy to understand

## Screenshots-
image

## Checklist for projects addition 
 
- [x] My project has contribution guidelines.
- [x] My project has a Code of Conduct.
- [x] My project has a README file.
- [x] My project has a License.

---

## Ready to `Merge` 👍🏻

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -2707,7 +2707,12 @@
         "link": "https://github.com/SyedImtiyaz-1/mini-Projects",
         "title": "JavaScript Practice Projects",
         "description": "JavaScript practice projects for the ones who want to learn JavaScript and this projects are beginner friendly.",
-        "tech": ["JavaScript, CSS, HTML, API's"]
+      },
+      {
+        "link": "https://github.com/SyedImtiyaz-1/calculator-",
+        "title": "This is a  Beginner Friendly Project for Students.",
+        "description": "JavaScript practice project for the ones who want to learn JavaScript and this projects are beginner friendly.",
+        "tech": ["JavaScript, CSS, HTML"]
```
Comment (priyankarpal) [inline src/DB/projects.json:2715]: change these 

```suggestion
        "tech": ["JavaScript", "CSS", "HTML"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
if you're a gssoc contributor make sure that don't create the same type of PR. if you create your pr might be invalid.

**Comment by SyedImtiyaz-1** [comment]
Sure @priyankarpal ,
it is ready to merge 👍🏻

File: src/DB/projects.json [hunk 2]
```diff
@@ -2713,7 +2713,12 @@
         "link": "https://github.com/SyedImtiyaz-1/mini-Projects",
         "title": "JavaScript Practice Projects",
         "description": "JavaScript practice projects for the ones who want to learn JavaScript and this projects are beginner friendly.",
-        "tech": ["JavaScript, CSS, HTML, API's"]
```
Comment (priyankarpal) [inline src/DB/projects.json:2716]: why did you remove the line?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by SyedImtiyaz-1** [comment]
@priyankarpal ,
now review the changes once 👍🏻

**Review by priyankarpal — APPROVED** [review]
